DancingGoat/Views/Shared/_Layout.cshtml

```diff
@@ -2,6 +2,11 @@
 @using DancingGoat.Controllers
 @using DancingGoat.Helpers
 @using DancingGoat.Models
+@using DancingGoat.Utils
+
+@functions {
+    public string ProjectGuid => ProjectUtils.GetProjectId();
```

Comment:
Footer view could get it on its own, but it is not a big deal, otherwise OK, no need to change this